src/Dropzone/assets/dist/controller.js

```diff
@@ -3,10 +3,16 @@ import { Controller } from '@hotwired/stimulus';
 class default_1 extends Controller {
     connect() {
         this.clear();
-        this.previewClearButtonTarget.addEventListener('click', () => this.clear());
-        this.inputTarget.addEventListener('change', (event) => this.onInputChange(event));
+        this.clear = this.clear.bind(this);
```

Comment:
Isn't it better to bind in `initialize` stage?

Reply:
Ah, yes, didn't think about that!

I'm never binding in connect and always use properties.

```js
class A {
    connect() {
        someElement.addEventListener('click', this._click);
    }

    _click = (event) => {
        …
    };
```